src/locales/default.json

```diff
@@ -97,9 +97,11 @@
     "learnMore": "Learn more",
     "light": "Light",
     "loading": "Loading...",
+    "liquidityMiningAPY": "Liquidity Mining APY",
     "loadMore": "Load more",
     "logOut": "Log out",
     "lpsEarnFee": "<span>LPs earn <b>{0}</b> in fees</span>",
+    "lpFeeAPY": "LP fees APY",
```

Comment:
We should change this to "Swap fees APY" - we use "swap fees" everywhere else in the app. Already cleared this with Pon.